chalice/app.py

```diff
@@ -17,10 +17,25 @@
 import requests
 from flask import json
 
-
 pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), 'chalicelib'))  # noqa
 sys.path.insert(0, pkg_root)  # noqa
 
+XRAY_TRACE = os.environ.get('XRAY_TRACE', 'false') == 'true'  # noqa
+
+if XRAY_TRACE:  # noqa
+    from aws_xray_sdk.core import xray_recorder, patch
+    from aws_xray_sdk.core.context import Context
+    from aws_xray_sdk.ext.flask.middleware import XRayMiddleware
+    patch(('boto3', 'requests'))
+    xray_recorder.configure(
+        service='DSS',
+        dynamic_naming='*dss.{}data.humancellatlas.org*'.format(
```

Comment:
why not just use `API_DOMAIN_NAME` here?

Reply:
Good idea, I didn't know about that